libs/platform/src/lib/table/components/table-cell-content/table-cell-content.component.ts

```diff
@@ -1,12 +1,11 @@
-import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+import { Component, Input, ViewEncapsulation } from '@angular/core';
 import { TableRow } from '../../models/table-row.model';
 import { TableColumn } from '../table-column/table-column';
 
 @Component({
     selector: 'fdp-table-cell-content',
     templateUrl: './table-cell-content.component.html',
-    encapsulation: ViewEncapsulation.None,
-    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
Default change detection may have performance impact, and since this is a cell component, it's very possible that the table will have a lots of these component instances. I would suggest keeping the OnPush strategy, and try using ngOnChanges to notify the component that important data has been changed